evalml/tests/data_checks_tests/test_invalid_targets_data_check.py

```diff
@@ -48,47 +48,6 @@ def test_invalid_target_data_check_numeric_binary_classification_valid_float():
     assert invalid_targets_check.validate(X, y) == {"warnings": [], "errors": [], "actions": []}
 
 
-def test_invalid_target_data_check_numeric_binary_classification_error():
```

Comment:
nitpick, but Is it possible to leave this in, but rather than checking for a warning, check that there isn't a warning thrown/the resulting warning list is empty? Just to ensure we have coverage that this case is supported